kafkakewl-metrics/src/main/scala/com/mwam/kafkakewl/metrics/services/HighOffsetMetricsExposer.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Marshall Wace <opensource@mwam.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.mwam.kafkakewl.metrics.services
+
+import com.mwam.kafkakewl.metrics.MainConfig
+import com.mwam.kafkakewl.metrics.domain.{KafkaTopicPartition, KafkaTopicPartitionInfo, KafkaTopicPartitionInfoChanges}
+import zio.kafka.consumer.Consumer
+import zio.{Duration, Fiber, Schedule, Task, UIO, URIO, ZIO, ZLayer}
+import zio.stream.{ZSink, ZStream}
+import zio.metrics.*
+import zio.metrics.Metric.Counter
```

Comment:
There are quite a few unused imports. If you build with sbt, you should see, or you can configure bsp in intellij and then it shows there too.